android-sdk/src/main/java/webtrekk/android/sdk/WebtrekkWebInterface.kt

```diff
@@ -61,15 +63,23 @@ open class WebtrekkWebInterface(private val webtrekk: Webtrekk) {
      */
     @JavascriptInterface
     fun trackCustomPage(pageName: String, params: String) {
-        // todo parse params to map and call webtrekk.trackCustomPage
+        try {
+            webtrekk.trackCustomPage(pageName, params.jsonToMap())
+        } catch (e: Exception) {
+            webtrekkLogger.info(e.message ?: "Something is a wrong")
```

Comment:
A more descriptive message would be better, maybe like "Unknown exception caught in webview while tracking custom page".